### PR TITLE
Add version and latest commit id to ONNX Runtime Server

### PR DESCRIPTION
**Pull request by NonStatic2014**

### CODE REVIEW

File: cmake/onnxruntime_server.cmake [hunk 1]
```diff
@@ -108,6 +108,14 @@ if(NOT WIN32)
   endif()
 endif()
 
+set(onnxruntime_SERVER_VERSION "local-build" CACHE STRING "Sever version")
+add_definitions(-DSRV_VERSION="${onnxruntime_SERVER_VERSION}")
+message(STATUS "ONNX Runtime Server version set to: ${onnxruntime_SERVER_VERSION}")
+
+set(onnxruntime_LATEST_COMMIT_ID "default" CACHE STRING "The latest commit id")
+add_definitions(-DLATEST_COMMIT_ID="${onnxruntime_LATEST_COMMIT_ID}")
```
Comment (snnn) [inline cmake/onnxruntime_server.cmake]: target_compile_definitions

**Comment by pranavsharma** [comment]
I checked in a fix for Linux CPU x64 NoContribops CI Pipeline today. can you please rebase?

**Pull request merged**